lib/contract/Contract.js

```diff
@@ -218,6 +218,15 @@ class Contract {
   hash() {
     return hash(this.serialize()).toString('hex');
   }
+
+  /**
+   * return
+   */
+  getDataTriggers() {
```

Comment:
we should be able to assign particular triggers to particular actions (`create`, `update`, `delete`) and document types.

Reply:
It's done in the DataTrigger class, we just haven't pushed it yet

Reply:
Do we still need it?

Reply:
It's done, I removed these params from here because of eslint. I can bring them back and add ignore the rule

Reply:
Let's remove it and use a factory in `executeDataTriggers`